### PR TITLE
Add support for `startFileId` for listFileVersions

### PR DESCRIPTION
**Pull request by ScottChapman**
Missing optional argument for startFileId

### CODE REVIEW

**Comment by ScottChapman** [comment]
resolves #79

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "backblaze-b2",
-  "version": "1.4.0",
+  "version": "1.4.1",
```
Comment (odensc) [inline package.json]: Looks good, but please keep the version the same as we have a process for releasing a new version.

Comment (ScottChapman) [reply package.json]: fixed

**Comment by odensc** [comment]
Oh, please add it to the README reference too.

**Comment by odensc** [comment]
Thanks for your contribution! #83

**Pull request merged**